### DISCUSSION TITLE
My fault interpolation not quite right?

### DISCUSSION THREAD

**Discussion by hangjiankiwi** [gempy-project/gempy#917] [Q&A] [answered]
Hi, I am trying to replicate a known structural geology from a dataset I made up.  These faults (named first to fourth in the pic attached) are parallel and stretching along the x axis. The interpolated faults however seem to do zig-zag which is not right. 

Much appreciated for any thoughts, thanks!
 
![Figure_1](https://github.com/gempy-project/gempy/assets/40050345/c925cdfc-844a-4583-aa00-bc36c01860f8)

**Comment by SimonVirgo** [comment, marked as answer]
At first glance I would assume your orientations are off by 90 degree. Did you use strike/dip instead of azimuth/dip connotation?

**Reply by hangjiankiwi** [reply]
Thanks Simon. yes, you are right, I just tested and I was off by 90 degrees.
